Content.Server/Salvage/SalvageSystem.Magnet.cs

```diff
@@ -45,7 +45,7 @@ private void OnMagnetClaim(EntityUid uid, SalvageMagnetComponent component, ref
             return;
         }
 
-        TakeMagnetOffer((station.Value, dataComp), args.Index, (uid, component));
+        _ = TakeMagnetOffer((station.Value, dataComp), args.Index, (uid, component));
```

Comment:
Discarding this task is not the right choice, it'll cause the error result to be poorly handled.

Reply:
Would catch-all-and-log suffice? Seems like it's too late to do anything else of value.

Reply:
> Would catch-all-and-log suffice? Seems like it's too late to do anything else of value.

Yeah, preferably.

Reply:
Great, done! https://github.com/space-wizards/space-station-14/pull/33794/commits/487edfc6d8cfb58d720f0d49da6adc1deb31cc51. Annotations there are fixed already, remaining red X is a spurious test failure ("system.activatable_u_i: Missing UI Key for entity: alien artifact (64056/n67622, MediumXenoArtifactItem)").